src/OrchardCore.Modules/OrchardCore.Users/Views/UserFields.cshtml

```diff
@@ -1,3 +1,9 @@
 @model SummaryAdminUserViewModel
-    
-<h5>@Model.User.UserName</h5>
\ No newline at end of file
+
+<h5 class="float-left">@Model.User.UserName</h5>
+@if (Model.User.IsDisabled)
+{
+    <span class="badge badge-danger ml-2" title="@T["Disabled"]">
+        <i class="fa fa-check" aria-hidden="true"></i> @T["Disabled"]
```

Comment:
https://fontawesome.com/icons/ban or https://fontawesome.com/icons/user-alt-slash could be appropriate.